alvr/adb/src/lib.rs

```diff
@@ -0,0 +1,142 @@
+// https://android.googlesource.com/platform/packages/modules/adb/+/refs/heads/main/docs/user/adb.1.md
+
+pub mod connection_state;
+pub mod device;
+pub mod forwarded_port;
+pub mod transport_type;
+
+use std::{io::Cursor, path::PathBuf, process::Command, time::Duration};
+
+use const_format::formatcp;
+use device::Device;
+use forwarded_port::ForwardedPort;
+use zip::ZipArchive;
+
+#[cfg(not(windows))]
+const ADB_EXECUTABLE: &str = "adb";
+#[cfg(windows)]
+const ADB_EXECUTABLE: &str = "adb.exe";
```

Comment:
We have `alvr_filesystem::exec_fname()` for this

Reply:
Macros increase compilation time. I don't see why you are concerned with having these as constants.

Reply:
As a rule of thumb, you don't have to use constants if you don't expect the value ever changing, just use inline literals. Only where the values are somewhat arbitrary, for example network ports or timeout durations, then use constants.